tools/expotools/src/commands/AndroidUnitTests.ts

```diff
@@ -0,0 +1,55 @@
+import spawnAsync from '@expo/spawn-async';
+import chalk from 'chalk';
+
+import * as Directories from '../Directories';
+import * as Packages from '../Packages';
+
+const ANDROID_DIR = Directories.getAndroidDir();
+
+const excludedInTests = [
+  "expo-module-template",
+  "expo-bluetooth",
+  "expo-notifications",
+  "expo-in-app-purchases",
+  "expo-updates"
+]
+
+async function action() {
+  const unimodules = await Packages.getListOfPackagesAsync();
+
+  function consoleErrorOutput(output: string, label: string, color: (string) => string): void {
+    const lines = output.trim().split(/\r\n?|\n/g);
+    console.error(lines.map(line => `${chalk.gray(label)} ${color(line)}`).join('\n'));
+  }
+
+  let androidPackages: string[] = [];
+  console.log(chalk.green('Unimodules to test: '));
+  for (const pkg of unimodules) {
+    const unimoduleName = pkg.unimoduleJson ? pkg.unimoduleJson.name : undefined;
+    if(unimoduleName && pkg.isSupportedOnPlatform('android') && !excludedInTests.includes(unimoduleName)) {
+      androidPackages = [...androidPackages, unimoduleName];
+      console.log(chalk.yellow(unimoduleName));
+    }
+  }
```

Comment:
```suggestion
  const androidPackages = unimodules
    .filter(unimodule => {
	  const unimoduleName = unimodule?.unimoduleJson.name;
	  return unimoduleName && unimodule.isSupportedOnPlatform('android') && !excludedInTests.includes(unimoduleName);
	})
	.map(unimodule => unimodule.unimoduleJson.name));
  console.log(chalk.green('Unimodules to test: '));
  androidPackages.forEach(unimoduleName => console.log(chalk.yellow(unimoduleName)));
```

Reply:
Why? Except for optional chaining, I don't get what's the purpose of suggestion?